### DISCUSSION TITLE
Is there Ubuntu 24 support?

### DISCUSSION THREAD

**Discussion by WarmWelcome** [JonathanTreffler/backblaze-personal-wine-container#155] [Q&A] [answered]
Just wondering if this is supported in ubuntu 24 LTS

**Comment by traktuner** [comment, marked as answer]
Hey @WarmWelcome!
I guess you mean ubuntu24 as base?
That will happen as soon as the ubuntu24 upstream base image is available. But it's definitely planned.

**Comment by WarmWelcome** [comment]
Yep. Thank you!
